docs/explanations/fly-scanning.md

```diff
@@ -0,0 +1,95 @@
+* Readback trigger: A hardware trigger sent when a signal from a sensor or encoder value is positively evaluated through some position compare logic.
+* Setpoint trigger: The same control signal sent to some device (e.g. a sample environment controller) also triggering the start of the collection window.
+
+### Scan frame triggers
+
+* Internal device trigger: The detector itself generates internal triggers based on its configuration.
+* Time trigger: Hardware triggers sent out at regularly spaced intervals in time.
+* Readback trigger: Same as for collection windows readback triggers.
+* Setpoint trigger: Same as for collection windows setpoint triggers.
+
+When a hardware trigger is sent to a detector, it can either be edge triggered (start a frame of a fixed duration on rising edge of trigger) or level triggered (start a frame on rising edge of trigger, stop the frame on falling edge).
+
+## Examples
+
+With all this in mind, let us consider typical fly scanning setups given a motor and a detector.
+
+### Software collection window internal triggers
+
+The simplest form of fly scanning uses software synchronisation for opening a collection window, and letting a detector trigger itself internally.
+We effectively tell the devices to "go" at the same time, relying on timestamps to match up the data. The strategy may involve
+
+* Calculating the motor run-up distance and move to the start, setting up the detector to take the right number of frames,
+* Starting the motor moving to its end point and waiting for the amount of time it takes to accelerate
+* Starting the detector
+* Waiting until the detector has finished taking frames and the motor has stopped moving
+
+This process would produce two data stream: a "monitor" stream of timestamped motor readback positions at the control system update rate;
+and a "primary" stream of detector frames with embedded timestamps when they were received by the control point.
+
+### Software collection window time triggers
+
+If we introduce a PandA box into the system we can trigger the collection window as above, but let the PandA produce time triggers for the detector.
+The strategy is similar:
+
+* Calculate motor run-up distance and move to the start, setting the detector into triggered mode and PandA to send the right number of triggers
+* Start the motor moving to its end point and wait for the amount of time it takes to accelerate
+* Start the PandA sending triggers
+* Wait until the PandA, detector and motor have finished
+
+The data streams produced are a "monitor" stream of timestamped motor readback positions at the control system update rate;
+and a "primary" stream of PandA captured timestamps and detector frames. If the motor happens to be plugged into the PandA
+then we can dispense with the monitor stream as the motor positions can be captured and produced by the PandA.
+
+### Readback collection window readback triggers
```

Comment:
I find these quite difficult to parse... Of course this is subjective, but to me just adding 'with' would make them much clearer e.g. Readback collection window with readback triggers